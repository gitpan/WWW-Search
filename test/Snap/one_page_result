http://home.snap.com/search/power/results/1,180,home-0,00.html?AM0=m&AN=1&AT0=w&DR=0&FD=1&FM=1&FR=f&KM=a&KW=%22WWW%3A%3ASearch%22%22Jim+Smyser%22&NR=100&PL=a
