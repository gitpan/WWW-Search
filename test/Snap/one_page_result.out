http://www.prweb.com/releases/1999/prweb7177.htm&
http://www.davecentral.com/8015.html&
http://www.isi.edu/lsam/tools/WWW_SEARCH/&
http://www.isi.edu/lsam/tools/WWW_SEARCH/index.html&
http://www.isi.edu/div7/lsam/tools/WWW_SEARCH/index.html&
http://www.isi.edu/div7/lsam/tools/WWW_SEARCH/index.html&
http://www.filelibrary.com/Contents/Multi-Platform/105/8.html&
http://www.isi.edu/~lsam/tools/WWW_SEARCH/index.html&
http://www.filelibrary.com/categories/New_Files.shtml3/35.html&
http://filelibrary.com/categories/New_Files.shtml3/32.html&
http://filelibrary.com/Contents/New_Files/150/34.html&
http://www.netpromote.com/search/WWW-Search-1.023/blib/man3/WWW::Search::Snap.3&
http://www.webreview.com/wr/pub/98/11/20/perl/index.html&
http://webreview.com/wr/pub/98/11/20/perl/index.html&
http://webreview.com/wr/pub/98/11/20/perl/index.html&
http://webreview.com/wr/pubwr/98/11/20/perl/index.html&
http://www.netpromote.com/search/WWW-Search-1.023/&
http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/README.html&
http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/&
http://ftp.epix.net/CPAN/authors/John_Heidemann/WWW-Search-1.023.readme&
http://ftp.sv.saitama-u.ac.jp/archives/lang/perl/CPAN/authors/id/JOHNH/WWW-Search-1.023.readme&
http://ftp.epix.net/CPAN/authors/John_Heidemann/WWW-Search-1.022.readme&
http://ftp.epix.net/CPAN/authors/John_Heidemann/WWW-Search-1.021.readme&
http://ftp.sv.saitama-u.ac.jp/archives/lang/perl/CPAN/authors/id/JOHNH/WWW-Search-1.022.readme&
http://ftp.sv.saitama-u.ac.jp/archives/lang/perl/CPAN/authors/id/JOHNH/WWW-Search-1.021.readme&
http://www.netpromote.com/search/WWW-Search-1.023/test/test.pl&
http://installationsplus.com/download.htm&
http://installationsplus.com/download.htm&
http://www.installationsplus.com/download.htm&
http://freeware2000.click2site.com/search_tools.htm&
http://www.pns.com.au/freewarehome/add_ons/a_internet.html&
http://www.inet.com.pl/freewarehome/add_ons/a_internet.html&
http://noguska.net/freewarehome/add_ons/a_internet.html&
http://ensignpub.com/freehome/add_ons/a_internet.html&
http://www3.winfiles.com/apps/98/webtools-search.html&
http://www.winfiles.com/apps/98/webtools-search.html&
http://www3.windows95.com/apps/98/webtools-search.html&
http://www1.windows95.com/apps/98/webtools-search.html&
http://www.winfiles.com/apps/98/webtools-search.html&
http://207.159.129.100/apps/98/webtools-search.html&
http://www1.winfiles.com/apps/98/webtools-search.html&
http://www2.windows95.com/apps/98/webtools-search.html&
http://winfiles.com/apps/98/webtools-search.html&
http://www.bookcase.com/library/software/win9x.winsock.www.html&
